src/components/autocomplete/js/autocompleteController.js

```diff
@@ -638,11 +639,13 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
       handleResults(items);
     } else if (items) {
       setLoading(true);
+      promiseFetch = true;
```

Comment:
Just for PR reviewer's, using the state of `setLoading` won't work, because it will be called anywhere else.